### DISCUSSION TITLE
Cookie consent sharing between different domains

### DISCUSSION THREAD

**Discussion by mikodayo** [orestbida/cookieconsent#531] [Q&A] [answered]
Hello. Thank you for developing this wonderful tool.
Could you please tell me one thing?
It is about sharing cookie consent between domains.

I understand that it is possible to share cookie consent between subdomains such as a.test.com, b.test.com,

However,  is it possible to share cookie consent between different domains such as test.com, test_1.com, etc.?
Could you please let me know if there is any way to do this?

**Comment by orestbida** [comment, marked as answer]
Hi @mikodayo, unfortunately this is not possible. 

You'd need a proper CMP that offers this feature, or implement your own custom API (which requires a  database to store the current consent state).

**Reply by mikodayo** [reply]
Hi @orestbida, Thank you for sharing the details.
I thought it was problematic to treat things between different domains as the same service, even though that is what the current situation is.
Therefore, I decided to operate with a subdomain attached to B.com.

```
A.com.
B.com (alias B.A.com )
```

Your service helped me a lot. Thank you very much.
